twilight-http/src/routing.rs

```diff
@@ -1650,6 +1702,27 @@ impl Display for Route<'_> {
 
                 f.write_str("/commands")
             }
+            Route::DeleteAutoModerationRule {
+                auto_moderation_rule_id,
+                guild_id,
+                ..
```

Comment:
Either drop the `..` or add it to `CreateAutoModerationRule` and `GetGuildAutoModerationRules`
```suggestion
                guild_id
```